hudi-common/src/main/java/org/apache/hudi/exception/HoodieIOException.java

```diff
@@ -18,27 +18,25 @@
 
 package org.apache.hudi.exception;
 
-import java.io.IOException;
```

Comment:
I don't think we should change the semantics of `HoodieIOException`. The idea is that an IOException is wrapped by hoodie to throw a HoodieIOException. This change gives the false impression by wrapping a generic exception.

Reply:
make sense